File: c/fahrenheit_to_celsius.c
#include <stdio.h>

int main()
{
    float fahrenheit_temperature = 0;
    printf("Digite uma temperatura em Farenheit: ");
    scanf("%f", &fahrenheit_temperature);
    float celsius_temperature = (fahrenheit_temperature - 32) * 5 / 9;
    printf("A temperatura em Celius é %.1f", celsius_temperature);
}

File: go/fahrenheit_to_celsius.go
package main

import "fmt"

func main() {
    var fahrenheit_temp float64

    fmt.Print("Digite uma temperatura em Fahrenheit: ")
    fmt.Scan(&fahrenheit_temp)
    
    var celsius_temp float64 = (fahrenheit_temp - 32) * 5 / 9;
    fmt.Printf("A temperatura em graus Celsius eh %.1fC", celsius_temp)
}